repo2docker/buildpacks/base.py

```diff
@@ -518,14 +554,25 @@ def get_assemble_scripts(self):
             ))
         except FileNotFoundError:
             pass
+        if self.stencila_contexts:
```

Comment:
not needed as `{'a', 'b'}.intersection(set())` works

Reply:
Fixed.